src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/RecallController.kt

```diff
@@ -262,6 +265,23 @@ class RecallController(
     createdDateTime
   )
 
+  fun RescindRecord.toResponse(documents: Set<Document>) = Api.RescindRecord(
+    id(),
+    version,
```

Comment:
We have version for Document & MDR, this feels similar to that, whereby you kind of care about the latest each time, tho want to see a history. Index we chose for address because it is a list that co-exist?

Reply:
Yeah; sounds right.  👍